.chloggen/add_podman_metadata.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: podmanreceiver
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: "Adds metrics and resources metadata and fixes invalid mapping of networking input and output metrics"
```

Comment:
It's unclear from the description what metrics and metadata are added. Can we make the migration to metadata.yaml as a no-op change for the end user and handle any other changes separately?

Reply:
Make sense to me, let's make the "networking" changes into another PR. Removed in: https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/30232/commits/91e24ff4bbbf8d199ab88b943c887781d27672c1